src/components/organisms/CesiumViewer.vue

```diff
@@ -0,0 +1,97 @@
+
+<script lang="ts">
+
+import { defineComponent, ref, onMounted, PropType, Ref } from 'vue';
+import useCesium from '@/store/cesium/useCesium';
+import { Location, Footprint } from '@/store/types';
+
+
+
+export default defineComponent({
+  name: 'CesiumViewer',
+  props: {
+    initialDestination: {
+      type: Object as PropType<Location>,
+      required: false,
+      default:() => ({
+        'longitude': -93.849688,
+        'latitude': 40.690265,
+        'height': 4000000
+      })
+    },
+    addFootPrint: {
+      type: Object as PropType<Footprint>,
+      required: false,
+    },
+    removeFootPrint: {
+      type: Number,
+      required: false,
+    }
+  },
+
+  setup(props){
+    const map: Ref<null | HTMLElement> = ref(null);
+
+    onMounted(async() =>{
+
+      const {
+        setDestination,
+        addFootprint,
+        removeFootprint,
+      } = useCesium(map);
+
+      if(props.initialDestination.longitude){
+      setDestination(props.initialDestination);
+      }
+
+      if(props.addFootPrint?.geoFootPrint){
+        addFootprint(props.addFootPrint.spatialID, props.addFootPrint.geoFootPrint);
+      }
+
+      if(props.removeFootPrint){
+        removeFootprint(props.removeFootPrint,);
+      }
+
+
+      });
```

Comment:
This block of code is slightly hard to read at first glance, mostly because the parentheses for the `onMounted` function aren't aligned. I'm surprised the linter didn't catch this, I think it may need to be configured more.

Reply:
You are likely right, I'll check out the configuration. Thanks!

Reply:
Still having some config issues but  this specific issue should be fixed with 38eb4c6fb645448f9e4199772d46bfbca8cac6fa

Reply:
:+1: I think we should have an issue to fix this - maybe even upstream in https://github.com/girder/vue-project-template since that also seems to be missing the formatting rules.